models/utils.py

```diff
@@ -10,7 +10,7 @@
 from errors import *
 
 
-def create_task(task):
+def create_task(user_id, task_id, function_id, endpoint_id):
```

Comment:
The function name create_task is a bit confusing.

Reply:
Changed this to log_invocation